framework/include/auxkernels/AuxKernel.h

```diff
@@ -119,37 +119,40 @@ class AuxKernelTempl : public MooseObject,
   const MaterialProperty<T> & getMaterialPropertyOlder(const std::string & name);
 
   template <typename T>
-  const T & getUserObject(const UserObjectName & name);
+  const T & getUserObject(const std::string & param_name) const;
```

Comment:
:+1: for this and the others

Reply:
There's still lots to be done for the PP and VPP interfaces. Let me make an issue...